buttercup/cogs/heatmap.py

```diff
@@ -59,7 +61,9 @@ def create_file_from_heatmap(
 
     timezone = utc_offset_to_str(utc_offset)
 
-    plt.title(i18n["heatmap"]["plot_title"].format(user=get_username(user)))
+    plt.title(
+        i18n["heatmap"]["plot_title"].format(user=get_username(user, escape=False))

```

Comment:
Question: escape=false. I tried to understand why this was added here, but I'm a bit stumped. (sorry if extremely silly)

Reply:
`escape` is being passed in as an argument to `get_username`, so we can follow the call to see what it's doing.

https://github.com/GrafeasGroup/buttercup/blob/7ef452c55beb4d7878e3318408937c75238c524d/buttercup/cogs/helpers.py#L235-L246

Here, it looks like `escape` calls `escape_formatting` here:

https://github.com/GrafeasGroup/buttercup/blob/7ef452c55beb4d7878e3318408937c75238c524d/buttercup/cogs/helpers.py#L136-L138

My guess is that since we're passing these usernames back to Discord instead of doing more work on them, we want to keep the formatting that Discord requires.

Reply:
Also, no such thing as a silly question here!

Reply:
The `escape` function is used to prevent characters in usernames to effect the formatting in Discord. For example, a username like `_ExampleUser_` would actually display as "_ExampleUser_" if we don't do this.

In this case though, we are putting the username into an image (the heatmap). Here, we don't have markdown formatting, so if we escaped the name it would show up as "\\\_ExampleUser\\\_".